### DISCUSSION TITLE
Google Assistant / App Actions

### DISCUSSION THREAD

**Discussion by Bob4321234** [libre-tube/LibreTube#1133] [Ideas]
Add ability to launch app and/or search/play media using Google Assistant. 

I'm not clear on the feasibility of this or if it conflicts with the principles behind keeping one's activity private from Google.

**Comment by Bnyro** [comment]
Google Assistant is closed source, therefore it is definitely against our principles and goals.

However, one thing that could be considered instead in the far future is integrating with [Dicio](https://github.com/Stypox/dicio-android), but it's not imaginable at the current state of the project.

**Reply by Bob4321234** [reply]
Got it, thanks for the quick response to this.
